comps/comps-foreman-plugins-el9.xml

```diff
@@ -0,0 +1,378 @@
+<?xml version="1.0"?>
+<!DOCTYPE comps PUBLIC "-//Red Hat, Inc.//DTD Comps info//EN" "comps.dtd">
+<comps>
+  <group>
+    <id>foreman-plugins</id>
+    <name>Foreman plugins</name>
+    <description>Plugin packages for the Foreman</description>
+    <uservisible>true</uservisible>
+    <packagelist>
+      <packagereq type="default">ansible-collection-theforeman-foreman</packagereq>
+      <packagereq type="default">ansible-collection-theforeman-operations</packagereq>
+      <packagereq type="default">ansible-runner</packagereq>
+      <packagereq type="default">ansiblerole-foreman_scap_client</packagereq>
+      <packagereq type="default">ansiblerole-insights-client</packagereq>
+      <packagereq type="default">cjson</packagereq>
+      <packagereq type="default">cjson-devel</packagereq>
+      <packagereq type="default">foreman-discovery-image-service</packagereq>
+      <packagereq type="default">foreman-discovery-image-service-tui</packagereq>
+      <packagereq type="default">mosquitto</packagereq>
+      <packagereq type="default">nodejs-react-json-tree</packagereq>
+      <packagereq type="default">puppet-foreman_scap_client</packagereq>
+      <packagereq type="default">puppetlabs-stdlib</packagereq>
+      <packagereq type="default">python3.11-ansible-runner</packagereq>
+      <packagereq type="default">python3.11-daemon</packagereq>
+      <packagereq type="default">python3.11-docutils</packagereq>
+      <packagereq type="default">python3.11-lockfile</packagereq>
+      <packagereq type="default">python3.11-packaging</packagereq>
+      <packagereq type="default">python3.11-pbr</packagereq>
+      <packagereq type="default">python3.11-pexpect</packagereq>
+      <packagereq type="default">python3.11-ptyprocess</packagereq>
+      <packagereq type="default">python3.11-pyparsing</packagereq>
```

Comment:
Will we have `python3.11` or just `python3`?

Reply:
Yes.
(I was trying to avoid this particular edge of this whole thing, as I'd prefer to use system python, but pulpcore might force our hand here)